demos/include/iot_demo_runner.h

```diff
@@ -38,7 +38,17 @@
  */
 
 /* Individual demo task entry definitions */
-#if defined( CONFIG_MQTT_DEMO_ENABLED )
+#if defined( CONFIG_PROVISIONING_DEMO_ENABLED )
+#define DEMO_entryFUNCTION                  RunProvisionDemo
+    #if defined( democonfigMQTT_ECHO_TASK_STACK_SIZE )
```

Comment:
Could provisioning have a tighter default stack size?

Reply:
Yes I think it could be much smaller, will do some testing if we go forward with this implementation